src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1457,5 +1457,10 @@ internal enum ErrorCode
         #endregion more stragglers for C# 7
 
         ERR_BadParameterModifiers = 8205,
+
+        ERR_ByRefConditionalAndAwait = 8301,
+        ERR_RefConditionalNeedsTwoRefs = 8302,
+        ERR_RefConditionalDifferentTypes = 8332,
```

Comment:
This is intentional to make it easier to merge from other branches that might also add errors. Will add a 'PRTOTYPE' comment to eventually make err ids contiguous

---
In reply to: [105267774](https://github.com/dotnet/roslyn/pull/17436#discussion_r105267774) [](ancestors = 105267774)